spec/unit/webrtc/call.spec.ts

```diff
@@ -485,25 +445,22 @@ describe('Call', function() {
                         return opponentMember;
                     }
                 },
-            };
+            } as unknown as Room;
         };
 
         const opponentCaps = {
             "m.call.transferee": true,
             "m.call.dtmf": false,
         };
-        call.chooseOpponent({
-            getContent: () => ({
-                version: 1,
-                party_id: "party_id",
-                capabilities: opponentCaps,
-            }),
-            getSender: () => opponentMember.userId,
-        });
+        (call as any).chooseOpponent(makeMockEvent(opponentMember.userId, {
+            version: 1,
+            party_id: "party_id",
+            capabilities: opponentCaps,
+        }));
 
         expect(call.getOpponentMember()).toBe(opponentMember);
-        expect(call.opponentPartyId).toBe("party_id");
-        expect(call.opponentCaps).toBe(opponentCaps);
+        expect((call as any).opponentPartyId).toBe("party_id");
```

Comment:
Won't this be on the `opponentMember`?

Reply:
opponentMember is a RoomMember which is just the Matrix member in the room rather than anything to do with the call, so no party ID?

Reply:
Ah, true, I was going on the assumption we use the `deviceId` for the `partyId`